jest/async-storage-mock.js

```diff
@@ -34,7 +34,7 @@ const asMock = {
   multiSet: jest.fn(_multiSet),
   multiRemove: jest.fn(_multiRemove),
   multiMerge: jest.fn(_multiMerge),
-  useAsyncStorage: jest.fn(key => {
+  useAsyncStorage: jest.fn((key) => {
```

Comment:
This seems like our Eslint did not keep in

Reply:
See https://github.com/react-native-async-storage/async-storage/pull/467#discussion_r511402295